rpcs3/util/vm_native.cpp

```diff
@@ -221,19 +221,23 @@ namespace utils
 	void memory_protect(void* pointer, usz size, protection prot)
 	{
 #ifdef _WIN32
-		for (u64 addr = reinterpret_cast<u64>(pointer), end = addr + size; addr < end;)
-		{
-			const u64 boundary = (addr + 0x10000) & -0x10000;
-			const u64 block_size = std::min(boundary, end) - addr;
 
-			DWORD old;
-			if (!::VirtualProtect(reinterpret_cast<LPVOID>(addr), block_size, +prot, &old))
+		DWORD old;
+		if (!::VirtualProtect(pointer, size, +prot, &old))
```

Comment:
```suggestion
	if (::VirtualProtect(pointer, size, +prot, &old))
		return;
```
Would be more correct to keep from moving unchanged code.